Mammoth/Include/TSE.h

```diff
@@ -783,7 +784,11 @@ class CSpaceObject
 		bool IsEnemy (const CDamageSource &Obj) const;
 		bool IsEnemyInRange (Metric rMaxRange, bool bIncludeStations = false);
 		bool IsEscortingFriendOf (const CSpaceObject *pObj) const;
+		bool IsEscorting (const CSpaceObject* pObj) const;
```

Comment:
If pObj is required, then make this a reference: 

bool IsEscorting (const CSpaceObject& Obj) const;

Otherwise, if it's optional (may be NULL), then it's OK.

I know much of the code doesn't follow this rule, but I'm trying to move towards that (newer) style.

Reply:
NULL is allowed by this code